app/backend/wells/filters.py

```diff
@@ -503,3 +516,113 @@ def get_filterset(self, request, queryset, view):
             filterset_class = WellListAdminFilter
 
         return filterset_class(**filterset_kwargs)
+
+    def filter_queryset(self, request, queryset, view):
+        filtered_queryset = super().filter_queryset(request, queryset, view)
+
+        filter_groups = request.query_params.getlist('filter_group', [])
+        for group in filter_groups:
+            try:
+                group_params = json.loads(group)
+            except ValueError:
+                # We ignore malformed JSON, so it doesn't break the request
```

Comment:
I think you can make an argument either way. I've gone with that the GET params shouldn't break the request, but it may be better to return a 400 in this case as you suggest, as if you're expecting a filtered response, and silently get an unfiltered one, that's not great. If you prefer a 400 let me know and I'll revise.

Reply:
We could get @stephenhillier in as a tie breaker - but I'd rather have the search fail, than not understand why it's not coming back with the expected results. If we do stay with not breaking the GET, we should at least do a logger.info / logger.warn on the exception.

Reply:
No, that's fine — I think you're right, failing obviously is better in this case. I'll revise.

Reply:
Updated in b4e6f99.